Sources/MockoloFramework/Utils/SwiftSyntaxExtensions.swift

```diff
@@ -0,0 +1,352 @@
+//
+//  SwiftSyntaxExtensions.swift
+//  MockoloFramework
+//
+//  Created by Ellie Shin on 10/29/19.
+//
+
+import Foundation
+import SwiftSyntax
+
+extension Trivia {
+    private func metadata(with annotation: String, in val: String) -> AnnotationMetadata? {
+        if val.contains(annotation) {
+            var aliasMap: [String: String]?
+            let comps = val.components(separatedBy: annotation)
+            if var last = comps.last, !last.isEmpty {
+                if last.hasPrefix("(") {
+                    last.removeFirst()
+                }
+                if last.hasSuffix(")") {
+                    last.removeLast()
+                }
+                if let aliaseArg = last.components(separatedBy: String.typealiasColon).last, !aliaseArg.isEmpty {
+                    let aliases = aliaseArg.components(separatedBy: String.annotationArgDelimiter)
+                    aliasMap = [String: String]()
+                    aliases.forEach { (item: String) in
+                        let keyVal = item.components(separatedBy: "=").map{$0.trimmingCharacters(in: CharacterSet.whitespaces)}
+                        if let k = keyVal.first, let v = keyVal.last {
+                            aliasMap?[k] = v
+                        }
+                    }
+                }
+            }
+            return AnnotationMetadata(typealiases: aliasMap)
+        }
+        return nil
+    }
+    func annotationMetadata(with annotation: String) -> AnnotationMetadata? {
+        var ret: AnnotationMetadata?
+        for i in 0..<count {
+            let trivia = self[i]
+            switch trivia {
```

Comment:
given similar functionality for `.docLineComment` and `.docBlockComment`, maybe this can be an `if` condition instead of a `switch`.

Reply:
moved this file to a separate PR